index.html

```diff
@@ -13107,13 +13107,27 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
         <div class="property" id="aria-controls">
           <pdef>aria-controls</pdef>
           <div class="property-description">
-            <p><a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the current element. See related <pref>aria-owns</pref>.</p>
-            <p>For example:</p>
+            <p>
+              <a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the focused element or composite widget. See related <pref>aria-details</pref> and
+              <pref>aria-owns</pref>.
+            </p>
+            <p>
+              The <code>aria-controls</code> property is for referencing elements that are modified by the user interacting with the currently focused element or composite widget.
+              The presence of <code>aria-controls</code> enables <a>assistive technologies</a> to make users aware of the current element’s association with the controlled element or elements. It can also inform them that an
+              association exists for instances where the controlled element was previously in the hidden state and through user interaction of the controlling element, the associated element has
+              become accessible.
+            </p>
+            <p>Instance where an <code>aria-controls</code> association could be made:</p>
             <ul>
-              <li>A table of contents tree view can control the content of a neighboring document pane.</li>
+              <li>A tree view representing a table of contents where choosing a treeitem updates content of a neighboring document pane.</li>
               <li>A group of checkboxes can control what commodity prices are tracked live in a table or graph.</li>
               <li>A tab controls the display of its associated tab panel.</li>
```

Comment:
as there is still mixed feelings about adding aria-expanded to the radio role - maybe this example better plays to what https://github.com/w3c/aria/issues/1404 is asking for.  e.g., since the radio cannot have its state toggled without making another selection - the expanded state is a bit awkward.  But, like the tab to tabpanel association example referenced here, that's arguably what people have been looking for - a programmatic way to convey that other content has rendered due to selection (or checking in the radio's case)

```suggestion
              <li>An interactive element reveals associated content when selected. For instance, selecting a tab control reveals its associated tab panel. Or checking a radio button reveals additional information or form controls related to the chosen radio button.</li>
```